services/src/layers/postgres_layer_db.rs

```diff
@@ -663,7 +660,7 @@ where
             workflow: serde_json::from_value(row.get(2)).context(crate::error::SerdeJson)?,
             symbology: row.get(3),
             properties: row.get(4),
-            metadata: serde_json::from_value(row.get(5)).context(crate::error::SerdeJson)?,
+            metadata: row.get::<_, HashMapTextTextDbType>(5).into(),
```

Comment:
why no delegate here?

Reply:
I cannot delegate to HashMap (foreign type) and I didn't want to change the field type in the struct.